src/data/zcl_abapgit_data_utils.clas.abap

```diff
@@ -168,6 +168,11 @@ CLASS zcl_abapgit_data_utils IMPLEMENTATION.
         lv_contflag = <ls_any>.
       CATCH cx_sy_dyn_call_illegal_class.
```

Comment:
Isn't one catch sufficient?
```abap
CATCH cx_sy_dyn_call_illegal_class cx_no_check.
  SELECT SINGLE contflag FROM ('DD02L') INTO lv_contflag WHERE tabname = iv_name.
```

Reply:
ahh, yea @gardian12 can you move the catch so there is only one?

Reply:
👍
Oh, yes. Wasn't aware of listen up the exceptions at one catch.
And yes, this is the better approach not to duplicate the code inside catch block.

I'll do the change.